.travis.yml

```diff
@@ -48,6 +48,18 @@ script:
   - make DESTDIR=`pwd`/deploy install
   - popd
 
+  - cd "${TRAVIS_BUILD_DIR}/doc"
```

Comment:
Better use `pushd` and call `popd` after line 53. That makes it extensible for the future without touching this 3 lines of code.